Dockerfile

```diff
@@ -28,20 +33,21 @@ EXPOSE 5000
 WORKDIR /code
 
 # NOTE: requirements.txt not likely to change between dev builds
-COPY requirements.txt /code/requirements.txt
+COPY requirements.txt .
 RUN apt-get update && \
     apt-get install -y --no-install-recommends netcat vim-tiny jq python3-dev xmlsec1 cron && \
     apt-get clean -y && \
     pip install -r requirements.txt
 
 # copy built react and node libraries for production mode
-COPY --from=node-prod-deps /usr/src/app/package-lock.json /code/package-lock.json
-COPY --from=node-prod-deps /usr/src/app/webpack-stats.json /code/webpack-stats.json
-COPY --from=node-prod-deps /usr/src/app/assets /code/assets
-COPY --from=node-prod-deps /usr/src/app/node_modules /code/node_modules
+COPY --from=node-prod-deps /usr/src/app/package-lock.json package-lock.json
+COPY --from=node-prod-deps /usr/src/app/webpack-stats.json webpack-stats.json
+COPY --from=node-prod-deps /usr/src/app/assets assets
+COPY --from=node-prod-deps /usr/src/app/node_modules node_modules
+COPY --from=node-prod-deps /usr/src/app/all_symlinks.tgz .
 
 # NOTE: project files likely to change between dev builds
-COPY . /code/
+COPY . .
```

Comment:
Couple questions here: 1) What made this change necessary? and 2) Is this line even necessary now with the change?

Reply:
This was kind of a separate issue and just a cleanup. Since the WORKDIR was /code it wasn't necessary to have /code/ everywhere. I can just remove it and put it in a separate issue.

Reply:
I think it's fine to keep this simplification in this issue. I guess I was looking for some clarity on why this line is necessary at all, if you're copying a directory contents to the same place.